src/main/java/org/elasticsearch/search/query/QuerySearchResult.java

```diff
@@ -114,6 +120,14 @@ public void aggregations(InternalAggregations aggregations) {
         this.aggregations = aggregations;
     }
 
+    public List<SiblingReducer> reducers() {
```

Comment:
I think it can only work for siblings since this is only concerned with top-level reducers and they must be sibling reducers since they have no parent?